common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4353,6 +4353,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
       new TimeValidator(TimeUnit.MILLISECONDS), "Collect llap daemon metrics in the AM every given milliseconds,\n" +
       "so that the AM can use this information, to make better scheduling decisions.\n" +
       "If it's set to 0, then the feature is disabled."),
+    LLAP_TASK_SCHEDULER_AM_COLLECT_DAEMON_METRICS_LISTENER("hive.llap.task.scheduler.am.collect.daemon.metrics.listener", "",
```

Comment:
Would remove the reference to AM here as we will change that implementation detail in the next stage.

Reply:
Since all of the related configs currently has AM in the names, I would keep it this way and when we move this out then we can rename them as needed.